### PR TITLE
PR #136 follow-up - add test for Helm values files

### LINKED ISSUE #93: Helm support for single or multiple values files
##### SUMMARY

Could be interesting that the helm module will support single or multiple values files. 
Helm command itself support it adding "-f values.yaml -f values2.yaml"  
##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

* helm

##### ADDITIONAL INFORMATION

```yaml

```

### PR DESCRIPTION
**Pull request by geerlingguy**
##### SUMMARY

Closes #136. Fixes #93.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

helm

##### ADDITIONAL INFORMATION

N/A

### CODE REVIEW

**Comment by fabianvf** [comment]
@geerlingguy does https://github.com/ansible-collections/community.kubernetes/pull/136#discussion_r452350040 still need to be addressed?

**Comment by geerlingguy** [comment]
@fabianvf - Yes. I'm not sure how to approach that since the files aren't guaranteed to be available locally, and they'd need to be translated from YAML to merge into the values dict.

**Comment by fabianvf** [comment]
@geerlingguy hmm, maybe we can just document that it may break idempotence for now?

**Comment by geerlingguy** [comment]
The idempotence issue may be fixed as part of adding support for `helm diff` later on, as part of #128.

**Review by maxamillion — APPROVED** [review]
LGTM so long as we have the follow-up to address the over-arching idempotent bits about helm in general (re: helmdiff discussion)

File: plugins/modules/helm.py [hunk 1]
```diff
@@ -73,6 +73,16 @@
...
+  values_files:
+    description:
+        - Value files to pass to chart.
+        - Paths will be read from the target host's filesystem, not the host running ansible.
+        - values_files option is evaluated before values option if both are used.
+        - Paths are evaluated in the order the paths are specified.
+    required: false
+    default: []
+    type: list
+    elements: str
```
Comment (Akasurde) [inline plugins/modules/helm.py:85]:
```suggestion
    elements: str
    version_added: '1.1.0'
```

**Review by Akasurde — APPROVED** [review]
LGTM

**Pull request merged**